server/sonar-server/src/test/java/org/sonar/server/authentication/UserIdentityAuthenticatorTest.java

```diff
@@ -332,7 +331,7 @@ public void authenticate_user_multiple_times_sets_root_flag_to_true_only_if_at_l
 
   @Test
   public void ignore_groups_on_non_default_organizations() throws Exception {
```

Comment:
@julienlancelot @sns-seb do we know how this feature will support organizations ?